src/app/pages/orders-page/orders-page.component.ts

```diff
@@ -1,8 +1,232 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { combineLatest, map, Observable, Subject, switchMap, take, takeUntil } from 'rxjs';
+import { formatDate } from '@angular/common';
+import {
+  selectCarriages,
+  selectOrders,
+  selectStations,
+  selectUsers,
+} from '../../core/store/trips/trips.selectors';
+import {
+  deleteOrder,
+  loadCarriages,
+  loadOrders,
+  loadStations,
+  loadUsers,
+} from '../../core/store/trips/trips.actions';
+import { Order } from '../../features/trips/models/order.model';
+import { selectAccess } from '../../core/store/user/user.selectors';
+import { Access } from '../../core/models/user.model';
+import { Carriage } from '../../features/trips/models/carriage.model';
+import { Station } from '../../features/trips/models/station.model';
+import { OrderForView } from './order/order.component';
+import { User } from '../../features/trips/models/user.model';
+
+interface CarriageData {
+  number: number;
+  type: string;
+  seat: number;
+}
 
 @Component({
   selector: 'app-orders-page',
   templateUrl: './orders-page.component.html',
-  styleUrl: './orders-page.component.scss',
+  styleUrls: ['./orders-page.component.scss'],
 })
-export class OrdersPageComponent {}
+export class OrdersPageComponent implements OnInit, OnDestroy {
+  orders$: Observable<Order[] | null> = this.store.select(selectOrders);
+
+  stations$: Observable<Station[] | null> = this.store.select(selectStations);
+
+  carriages$: Observable<Carriage[] | null> = this.store.select(selectCarriages);
+
```

Comment:
Please, don't forget about access modifiers

Reply:
I corrected it )